templates/CRM/Admin/Form/Preferences/Mailing.hlp

```diff
@@ -0,0 +1,18 @@
+{*
+ +--------------------------------------------------------------------+
+ | Copyright CiviCRM LLC. All rights reserved.                        |
+ |                                                                    |
+ | This work is published under the GNU AGPLv3 license with some      |
+ | permitted exceptions and without any warranty. For full license    |
+ | and copyright information, see https://civicrm.org/licensing       |
+ +--------------------------------------------------------------------+
+*}
+{htxt id="no_reply_email_address-title"}
+  {ts}Email From Address for no-reply emails.{/ts}
+{/htxt}
+{htxt id="no_reply_email_address"}
+  {capture assign=fromEmailOptionsURL}{crmURL p='civicrm/admin/options/from_email_address' q="reset=1"}{/capture} 
+  <p>{ts 1=$fromEmailOptionsURL}Specify an Email From Address to use when the system sends an email but a reply is not expected, for example when a user is sent an email for a double opt-in.{/ts}</p>
+  <p>{ts 1=$fromEmailOptionsURL}Leaving this blank will use the default which will be do-not-reply@default_domain where the default_domain will be the email domain address of your default mailing account also used for bounce handling.{/ts}</p>
+  <p>{ts 1=$fromEmailOptionsURL}You can add additional Email From Addresses <a href='%1'>here</a>.{/ts}</p>
```

Comment:
In general a single word like "here" or "click" doesn't make for a very self-explanatory link. How about the whole thing as a link. Then you don't need any variables in {ts} and you don't need any html in your translated string. (and can remove the `capture assign` line above too)
```suggestion
  <p><a href="{crmURL p='civicrm/admin/options/from_email_address' q="reset=1"}">{ts}Edit From Email Addresses{/ts}</a></p>
```